x/service/const.go

```diff
@@ -1,5 +1,23 @@
 package service
 
+import (
+	"github.com/pokt-network/pocket-core/crypto"
+	"github.com/pokt-network/pocket-core/tests/fixtures"
+)
+
 const (
+	// The http method defaults to POST for Relays
+	// because JSON RPC uses POST for all calls
 	DEFAULTHTTPMETHOD = "POST"
+	// HTTP Service Types
+	RPC ServiceType = iota + 1
+	REST
+
+	// TODO remove and get from world state
```

Comment:
This should get the app pub key from config, not the world state.

Reply:
Nope, app pub key is from the world state

Reply:
Node public key (self) is from config

Reply:
Got it, thanks for the clarification.